platform/wayland/cog-utils-wl.h

```diff
@@ -26,20 +26,6 @@ G_BEGIN_DECLS
 #define DEFAULT_HEIGHT 768
 #define DEFAULT_WIDTH  1024
 
-#if defined(WPE_CHECK_VERSION)
-#    define HAVE_REFRESH_RATE_HANDLING WPE_CHECK_VERSION(1, 13, 2)
-#else
-#    define HAVE_REFRESH_RATE_HANDLING 0
-#endif
-
-#if defined(WPE_FDO_CHECK_VERSION)
-#    define HAVE_SHM_EXPORTED_BUFFER WPE_FDO_CHECK_VERSION(1, 9, 0)
-#    define HAVE_FULLSCREEN_HANDLING WPE_FDO_CHECK_VERSION(1, 11, 1)
```

Comment:
I think you need to drop the support in the gtk4 platform too:

``` c
aperezdc/bump-libwpe psaavedra@golpe ~/local/git/WPEBackend-fdo $ grep -C 2 -n -r HAVE_FULLSCREEN_HANDLING  * 
platform/gtk4/cog-platform-gtk4.c-27-
platform/gtk4/cog-platform-gtk4.c-28-#if defined(WPE_FDO_CHECK_VERSION)
platform/gtk4/cog-platform-gtk4.c:29:#    define HAVE_FULLSCREEN_HANDLING WPE_FDO_CHECK_VERSION(1, 11, 1)
platform/gtk4/cog-platform-gtk4.c-30-#else
platform/gtk4/cog-platform-gtk4.c:31:#    define HAVE_FULLSCREEN_HANDLING 0
platform/gtk4/cog-platform-gtk4.c-32-#endif
platform/gtk4/cog-platform-gtk4.c-33-
--
platform/gtk4/cog-platform-gtk4.c-73-    int width;
platform/gtk4/cog-platform-gtk4.c-74-    int height;
platform/gtk4/cog-platform-gtk4.c:75:#if HAVE_FULLSCREEN_HANDLING
platform/gtk4/cog-platform-gtk4.c-76-    bool is_fullscreen;
platform/gtk4/cog-platform-gtk4.c-77-    bool waiting_fullscreen_notify;
--
platform/gtk4/cog-platform-gtk4.c-232-}
platform/gtk4/cog-platform-gtk4.c-233-
platform/gtk4/cog-platform-gtk4.c:234:#if HAVE_FULLSCREEN_HANDLING
platform/gtk4/cog-platform-gtk4.c-235-static void
platform/gtk4/cog-platform-gtk4.c-236-dispatch_wpe_fullscreen_event(struct platform_window* win)
--
platform/gtk4/cog-platform-gtk4.c-457-}
platform/gtk4/cog-platform-gtk4.c-458-
platform/gtk4/cog-platform-gtk4.c:459:#if HAVE_FULLSCREEN_HANDLING
platform/gtk4/cog-platform-gtk4.c-460-static bool
platform/gtk4/cog-platform-gtk4.c-461-on_dom_fullscreen_request(void* unused, bool fullscreen)
--
platform/gtk4/cog-platform-gtk4.c-574-    g_signal_connect(window->gl_drawing_area, "notify::scale-factor", G_CALLBACK(scale_factor_change), window);
platform/gtk4/cog-platform-gtk4.c-575-
platform/gtk4/cog-platform-gtk4.c:576:#if HAVE_FULLSCREEN_HANDLING
platform/gtk4/cog-platform-gtk4.c-577-    g_signal_connect(window->gtk_window, "notify::fullscreened", G_CALLBACK(on_fullscreen_change), window);
platform/gtk4/cog-platform-gtk4.c-578-#endif
--
platform/gtk4/cog-platform-gtk4.c-690-    cog_gamepad_setup(gamepad_provider_get_view_backend_for_gamepad);
platform/gtk4/cog-platform-gtk4.c-691-
platform/gtk4/cog-platform-gtk4.c:692:#if HAVE_FULLSCREEN_HANDLING
platform/gtk4/cog-platform-gtk4.c-693-    wpe_view_backend_set_fullscreen_handler(webkit_web_view_backend_get_wpe_backend(win.view_backend),
platform/gtk4/cog-platform-gtk4.c-694-                                            on_dom_fullscreen_request, NULL);
```

Reply:
This was from the commit in #686 - this PR has to be merged _after_. I have updated the other PR with the suggestion.